x-pack/legacy/plugins/uptime/public/components/functional/monitor_list/monitor_list.tsx

```diff
@@ -61,11 +60,14 @@ export const MonitorListComponent = (props: Props) => {
     linkParameters,
     loading,
   } = props;
+
   const [drawerIds, updateDrawerIds] = useState<string[]>([]);
-  const items = get<MonitorSummary[]>(data, 'monitorStates.summaries', []);
 
-  const nextPagePagination = get<string>(data, 'monitorStates.nextPagePagination');
-  const prevPagePagination = get<string>(data, 'monitorStates.prevPagePagination');
+  const items = data?.monitorStates?.summaries || [];
+  const nextPagePagination: string | undefined =
+    data?.monitorStates?.nextPagePagination ?? undefined;
+  const prevPagePagination: string | undefined =
+    data?.monitorStates?.prevPagePagination ?? undefined;
```

Comment:
why you have to assign undefined explicitly? it will be undefined is there is no value.

Reply:
Sorry, thought I answered this already - the type of `prevPagePagination` is `string | null | undefined`, so while an `undefined` value will still be `undefined`, it could equal `null`, which is why the explicit assignment.